### PR TITLE
chore: changes from org move

### PR DESCRIPTION
**Pull request by richardcase**
Changes as a result of moving the repo to the k3s org. This includes:

- Package names/imports
- Image names

### CODE REVIEW

**Comment by zawachte** [comment]
moving quick :)

File: Makefile [hunk 1]
```diff
@@ -110,7 +110,7 @@ $(RELEASE_DIR):
 $(RELEASE_NOTES_DIR):
 	mkdir -p $(RELEASE_NOTES_DIR)/
 
-REGISTRY ?= ghcr.io/cluster-api-provider-k3s/cluster-api-k3s
+REGISTRY ?= ghcr.io/k3s-io/cluster-api-k3s
```
Comment (zawachte) [inline Makefile:113]: unsure how the perms on this work for this org. Do you have any clue?

**Review by zawachte — APPROVED** [review]

**Comment by zawachte** [comment]
Going to merge this and we can figure out the container registry/release details after.

File: Makefile (hunk 1 above)
Comment (richardcase) [reply Makefile:113]: I think we should be good as the workflow specifies write packages.

I have update the docker file to label it from this new repo. Also, i tested that the old images are still available. So this worked:

```bash
docker pull ghcr.io/cluster-api-provider-k3s/cluster-api-k3s/bootstrap-controller:v0.1.9
```

**Pull request merged**